public/components/common/seach/autocomplete_service.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect, Fragment } from 'react';
+import {
+    EuiFieldSearch,
+    EuiFlexItem,
+    EuiListGroup,
+    EuiListGroupItem,
+    EuiText,
+    EuiSelectableTemplateSitewide,
+    EuiSelectableTemplateSitewideOption,
+    EuiLink,
+    EuiFlexGroup
+  } from '@elastic/eui';
+  import {
```

Comment:
not a problem but some indentation seems off, could you run a formatter over these files? i think we are mostly using prettier as formatter

Reply:
Will be getting rid of these files, but I've run eslint over the other ones